data/core.toml

```
@@ -16,27 +16,27 @@ advisory = [
 "Serhat Can",
 "Jennifer Davis",
 "Patrick Debois (founder, 2009–2014 lead)",
+"Yvo van Doorn (2021–2023 co-chair)",
 "Nathen Harvey",
 "Bridget Kromhout (2015–2020 lead)",
 "Ken Mugrage",
 "Matty Stratton (2021–2022 co-chair)",
```

Heh, this trailing comma is an issue
```suggestion
"Matty Stratton (2021–2022 co-chair)"
```

---

Last-line-of-a-list commas matter don't matter in TOML:

https://toml.io/en/v1.0.0#array

---

Huh, that's neat. 
I'm not sure I understand why it failed as we've had TOML without the trailing comma

Your solution of adding a trailing comma to all the arrays is a good one